workers/tests/integrationtests/test_indicator_tags_ratio.py

```diff
@@ -62,19 +62,6 @@ def test_no_features(self):
         self.assertEqual(indicator.result.label, "undefined")
         self.assertEqual(indicator.result.value, None)
 
-    @oqt_vcr.use_cassette()
-    def test_no_filter2(self):
-        """Layer with no filter2 for ratio endpoint"""
-        layer_name = "major_roads_length"
-        dataset = "regions"
-        feature_id = 2
-        bpolys = asyncio.run(db_client.get_bpolys_from_db(dataset, feature_id))
-
-        indicator = TagsRatio(layer_name=layer_name, bpolys=bpolys)
-        asyncio.run(indicator.preprocess())
-        self.assertIsNone(indicator.count_all)
-        indicator.calculate()
-
```

Comment:
Which test replaces this test? Is there another test checking if a ratio query is made without a `filter2` or `ratio_filter` argument?

Please remove the referring query in the cassettes file as well.

Reply:
The Tag Ratio indicator works only for layer with the `ratio_filter`. Since changes introduced in this PR indicator + layer combination are validated during initialization of the indicator object. This will cover up layer with not  `ratio_filter` for the Tag Ratio indicator.

For this particular test case a `IndicatorLayerError` will be raised before the test finishes.

Reply:
I recreated the caassette for this test.